cleanlab_studio/studio/studio.py

```diff
@@ -210,10 +214,22 @@ def create_project(
             api_key=self._api_key,
             dataset_id=dataset_id,
             project_name=project_name,
-            tasktype=tasktype,
+            task_type=task_type,
             modality=modality,
-            modeltype=modeltype,
+            model_type=model_type,
             label_column=label_column,
             feature_columns=feature_columns if feature_columns is not None else [],
             text_column=text_column,
         )
+
+    def poll_cleanset_status(self, cleanset_id: str, timeout: Optional[int] = None) -> bool:
```

Comment:
Even though it's already fairly clear, I would put in docstring that this operation will block until cleanset is ready, errored, or timeout has been reached.